src/odemis/driver/hamamatsurx.py

```diff
@@ -877,7 +877,7 @@ class StreakCamera(model.HwComponent):
     Client to connect to HPD-TA software via RemoteEx.
     """
 
-    def __init__(self, name, role, port, host, children=None, daemon=None, **kwargs):
+    def __init__(self, name, role, port, host, dependencies=None, daemon=None, **kwargs):
```

Comment:
As you've properly done in the YAML file for the streakcam, it needs both children and dependencies. Almost all the components are children, and the only one special is the "spectrograph", which is an (optional) dependency.